src/stan/analyze/mcmc/compute_potential_scale_reduction.hpp

```diff
@@ -35,6 +35,9 @@ inline double compute_potential_scale_reduction(
     std::vector<const double*> draws, std::vector<size_t> sizes) {
   int num_chains = sizes.size();
   size_t num_draws = sizes[0];
+  if (num_draws == 0) {
+    return std::numeric_limits<double>::quiet_NaN();
```

Comment:
@andrjohns, the change looks great.

Before it gets merge, just a question: is `quiet_NaN` the right behavior here? I don't know how this gets used next, so just wanting to sure this isn't going to trigger other problems. (I'm guessing that NaN is the right choice; just wanting to double check.)

Reply:
The only place this is used is for user-facing output (from `stansummary`) so I believe NaN is a reasonable choice. We also return NaN if all chains are constant a bit further down in the function